### PR TITLE
net-ftp/ncftp: bump to EAPI 8 and fix bugs

### PR DESCRIPTION
**Pull request by ceamac**
Rename `configure.in` to `configure.ac` and disable `strip`

### CODE REVIEW

File: net-ftp/ncftp/ncftp-3.2.6-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		econf \
+		$(use_enable pch precomp) \
+		--disable-ccdv \
+		--disable-universal
+}
+
+src_install() {
+	default
+
+	DOCS=( README.txt doc/*.txt )
```
Comment (Flowdalic) [inline net-ftp/ncftp/ncftp-3.2.6-r4.ebuild:53]: Sometimes  a glob in `DOCS` could cause issues,  see also https://github.com/gentoo/gentoo/pull/23668#issuecomment-1010143030. 

I recommend to use `dodoc` and `dohtml`, potentially with `-r`, instead.

Comment (ceamac) [reply net-ftp/ncftp/ncftp-3.2.6-r4.ebuild:53]: `DOCS` and `HTML_DOCS` with globbing cause problems if declared at global scope, but used inside a function they work fine.

But passing `-r` to `einstalldocs` does nothing, so I should change this anyway.

`dohtml` is banned in EAPI 7, and the suggestion [here](https://devmanual.gentoo.org/function-reference/install-functions/) is to switch to `einstalldocs`, that's why I tried this.  I also tried declaring `DOCS` and `HTML_DOCS` at global level without globbing, but I didn't like the result.

So I can either remove `-r` or simply revert to `docinto`/`dodoc` as in `ncftp-3.2.6-r3.ebuild`.  What do you think is better?

Comment (Flowdalic) [reply net-ftp/ncftp/ncftp-3.2.6-r4.ebuild:53]: I think

```
dodoc README.txt doc/*.txt
docinto html
dodoc doc/html/*.html
```

is just fine. One could maybe consider replacing the final `dodoc` with `dodoc -r doc/html`.

Comment (ceamac) [reply net-ftp/ncftp/ncftp-3.2.6-r4.ebuild]: Done.  Thank you

**Comment by Flowdalic** [comment]
Thanks for your contribution.

**Pull request closed without merging**